pandas/core/indexes/base.py

```diff
@@ -2971,9 +2971,9 @@ def _convert_listlike_indexer(self, keyarr, kind=None):
 
         Returns
         -------
-        tuple (indexer, keyarr)
+        indexer, keyarr
             indexer is an ndarray or None if cannot convert
-            keyarr are tuple-safe keys
+            keyarr are tuple-safe keys.
```

Comment:
do you mind documenting both parameters separately? It's done in other parts of this PR